src/librustc_resolve/late/diagnostics.rs

```diff
@@ -1084,6 +1086,15 @@ impl<'tcx> LifetimeContext<'_, 'tcx> {
                 }
             }
         }
+        if nightly_options::is_nightly_build()
+            && !self.tcx.features().in_band_lifetimes
+            && suggests_in_band
+        {
+            err.help(
+                "if you want to use in-band lifetime bindings, \
```

Comment:
```suggestion
                "if you want to experiment with in-band lifetime bindings, \
```

Not sure this is a feature that should ever be recommended.
I'd personally want to see it removed rather than stabilized.
I guess we can recommend it to get more experience, but the suggestion shouldn't look too encouraging.

Reply:
Thanks! Addressed via a9b6476.